java-sdk/src/main/java/com/adobe/target/edge/client/ClientConfig.java

```diff
@@ -33,6 +33,7 @@
     private boolean logRequests;
     private boolean logRequestStatus;
     private HttpRequestInterceptor requestInterceptor;
+    private ClientProxyConfig proxy;
```

Comment:
Let's rename the variable to `clientProxyConfig` or `proxyConfig`

Reply:
Renamed to proxyConfig, as part of commit (https://github.com/adobe/target-java-sdk/pull/6/commits/24a88c3a66894ebeb719418d4f95969a2b56e11c)